api/src/main/resources/swagger3codegen/libraries/okhttp-gson/GzipRequestInterceptor.mustache

```diff
@@ -0,0 +1,80 @@
+{{!
+
+This is an AoU-specific extension of the default Swagger-codegen okhttp API implementation template.
+
+The original template was copied from the 3.0.21 release of swagger-codegen; see
```

Comment:
These 4 files are strict copies from the swagger-api repo.  No changes in functionality are expected.

Reply:
except for these headers of course